core/src/main/scala/org/apache/spark/deploy/history/FsHistoryProvider.scala

```diff
@@ -43,7 +43,7 @@ import org.apache.spark.{Logging, SecurityManager, SparkConf}
 private[history] class FsHistoryProvider(conf: SparkConf) extends ApplicationHistoryProvider
   with Logging {
 
-  import FsHistoryProvider._
+  import org.apache.spark.deploy.history.FsHistoryProvider._
```

Comment:
nit: this seems like your IDE ran through this code, but this (and the other import changes) are not really necessary.